docs/configuring-metabase/caching.md

```diff
@@ -103,11 +105,26 @@ To set a caching policy for a question, you must have [curate access](../permiss
 4. Select the [caching invalidation policy](#cache-invalidation-policies).
 5. Save your changes.
 
+## How dashboard, question, database, and default caching policies interact
+
+If you have multiple caching policies set up, Metabase will use the first available policy, in this order:
```

Comment:
In my head we either "set" or "assign" a caching policy. I'm less sure of "set up".